public/locales/es/translation.json

```diff
@@ -1,15 +1,16 @@
 {
-  "subheaderButton": "Go to Benefit Programs Online",
-  "subheaderHeader": "Guide to applying for unemployment benefits",
-  "subheaderSubheader": "Learn what type of benefits you qualify for and how to apply for them.",
-  "subheaderParagraph": "If you are out of work or had your hours reduced you may be eligible to receive unemployment benefits from California’s Employment Development Department (EDD).",
+  "subheaderButton": "Ir a los Programas de Beneficios Online",
```

Comment:
It looks like we don't actually use this anywhere 😬 The BPO button component still has text within the component. Do you mind updating that?

Reply:
good catch!